Python/peephole.c

```diff
@@ -477,10 +502,11 @@ PyCode_Optimize(PyObject *code, PyObject* consts, PyObject *names,
                 break;
         }
         nexti = i - op_start + 1;
-        if (instrsize(j) > nexti)
+        if (instrsize(j) > nexti || (size_t)nexti > UINT_MAX) {
```

Comment:
`nexti` here should be a small integer in range 1-4.

Reply:
I modified my PR to use an assertion.